examples/solid-client-access-grants-demo/README.md

```diff
@@ -0,0 +1,49 @@
+# Solid Access Requests and Access Grants Demo
+
+## Run the demo app
+
+- Run `npm ci` and `npm run build` in the current directory to build the demo app.
```

Comment:
(/examples/solid-client-access-grants-demo/)
maybe add this as a side note after "current directory" so that if I'm in my terminal I don't need to go to the folder structure to see where I am

Reply:
Very good suggestion :) https://github.com/inrupt/solid-client-access-grants-js/pull/252/commits/8b4e1f37dee4cfca59daf3b7d89d966fd6cc6181